devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -303,75 +303,258 @@ bool WholeBodyDynamicsDevice::openEstimator(os::Searchable& config)
     return true;
 }
 
-bool WholeBodyDynamicsDevice::openDefaultContactFrames(os::Searchable& config)
+bool WholeBodyDynamicsDevice::openContactFrames(os::Searchable& config)
 {
     yarp::os::Property prop;
     prop.fromString(config.toString().c_str());
 
-    yarp::os::Bottle *propContactFrames=prop.find("defaultContactFrames").asList();
-    if(propContactFrames==0)
+    yarp::os::Bottle *propDefaultContactFrames=prop.find("defaultContactFrames").asList();
+    yarp::os::Bottle *propOverrideContactFrames=prop.find("overrideContactFrames").asList();
+    yarp::os::Bottle *propContactWrenchType=prop.find("contactWrenchType").asList();
+    yarp::os::Bottle *propContactWrenchDirection=prop.find("contactWrenchDirection").asList();
+    yarp::os::Bottle *propContactWrenchPosition=prop.find("contactWrenchPosition").asList();
+
+    if(propOverrideContactFrames==0)
     {
-       yError() <<"wholeBodyDynamics : Error parsing parameters: \"defaultContactFrames\" should be followed by a list\n";
-       return false;
+        //overrideContactFrames don't exist -> considering defaultContactFrames
+        if(propDefaultContactFrames==0)
+        {
+            yError() <<"wholeBodyDynamics : Error parsing parameters: \"defaultContactFrames\" should be followed by a list\n";
```

Comment:
Please consider adding the function names in these error messages.

Reply:
Done in https://github.com/robotology/whole-body-estimators/pull/48/commits/38c21cdf2a12d33f0fd9a3f736d69353bb89e86e for the messages in `openContactFrames()`. In general, the the messages in the rest of the device are don't have the name of the function. We may change it in a separate PR if necessary.